resolwe_bio/tools/parse_diffexp.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python2
+import argparse
+import json
+import pandas as pd
+
+
+parser = argparse.ArgumentParser(description="Parse Diff Exp output files")
+parser.add_argument('de_file', help="DE analysis output file (.tab).")
+args = parser.parse_args()
+
+de_data = pd.read_csv(args.de_file, sep='\t')
+de_data.rename(columns={'Unnamed: 0': 'gene_id'}, inplace=True)
+columns = {}
+
+for column in de_data:
+    columns[column] = list(de_data[column])
+
+data = {'de_data': columns}
```

Comment:
@nkran @lukaw3d This parses whole Diff Exp results table into separate columns and saves to storage. The idea is to build DE results tables and volcano plot in front-end from this results, and than allow for the changes to the plot and tables to be made from the settings (e.g. which column for the Y axis, what are the labels..). Do you propose some other structure for this JSON? For example, should it include default fields like fc, fdr, gene_ids, axis labels?